bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/service/ReadyMarker.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.service;
+
+/**
+ * Marker service in order to denote that loading of a specific component has completed and it is ready to be used.
+ *
+ * @author Simon Kaufmann - initial contribution and API
+ *
+ */
+public interface ReadyMarker {
+
+    public final static String XML_THING_TYPE = "esh.xmlThingTypes";
```

Comment:
I am not completely happy to define those constants in smarthome.core as this bundle in theory does not know anything about the existence of the xml bundles.